tools/estimate.wdl

```diff
@@ -5,6 +5,13 @@
 version 1.1
 
 task calc_tpm {
+    meta {
+        description: "Given a gene counts file and a gene lengths file, calculate Transcripts Per Million (TPM)"
+        outputs: {
+            tpm_file: "Transcripts Per Million (TPM) file. A two column headered TSV file."
+        }
+    }
```

Comment:
Good question. I think I'd rather see that sort of "help"/discussion in the header of the file, rather than as a long, poorly formatted, WDL string.

I have plans to at some point rewrite all our headers, IMO if we want to distinguish FPKM/RPKM/TPM it would belong there, not here.